VirusTree.java

```diff
@@ -330,8 +330,44 @@ public static void rotate() {
 				p.setTraitB(y);					
 			}
 
-		}	
-		
+		}
+
+		if (Parameters.phenotypeSpace.equals("geometricSeq")) {
```

Comment:
I forget, did you mostly just copy this code block or is it new code? If it's new, would you please document it a little more?